src/test/java/org/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -254,1144 +64,106 @@ public void setUp() throws IOException, COSVisitorException {
     }
 
     /**
-     * Most basic test for reading.
-     * @throws IOException
-     * @throws COSVisitorException
-     */
-    @Test
-    public void testReadXMPSimple() throws COSVisitorException, IOException {
-
-        String bibtex = "<bibtex:year>2003</bibtex:year>\n"
-                + "<bibtex:title>Beach sand convolution by surf-wave optimzation</bibtex:title>\n"
-                + "<bibtex:bibtexkey>OezbekC06</bibtex:bibtexkey>\n";
-
-        writeManually(pdfFile, XMPUtilTest.bibtexXPacket(XMPUtilTest.bibtexDescription(bibtex)));
-
-        List<BibEntry> l = XMPUtil.readXMP(pdfFile.getAbsoluteFile(), xmpPreferences);
-        Assert.assertEquals(1, l.size());
-        BibEntry e = l.get(0);
-
-        Assert.assertNotNull(e);
-        Assert.assertEquals(Optional.of("OezbekC06"), e.getCiteKeyOptional());
-        Assert.assertEquals(Optional.of("2003"), e.getField("year"));
-        Assert.assertEquals(Optional.of("Beach sand convolution by surf-wave optimzation"),
-                e.getField("title"));
-        Assert.assertEquals("misc", e.getType());
-    }
-
-    /**
-     * Is UTF8 handling working? This is because Java by default uses the platform encoding or a special UTF-kind.
-     * @throws IOException
-     * @throws COSVisitorException
-     */
-    @Test
-    public void testReadXMPUTF8() throws COSVisitorException, IOException {
-
-        String bibtex = "<bibtex:year>2003</bibtex:year>\n" + "<bibtex:title>�pt�mz�t��n</bibtex:title>\n"
-                + "<bibtex:bibtexkey>OezbekC06</bibtex:bibtexkey>\n";
-
-        writeManually(pdfFile, XMPUtilTest.bibtexXPacket(XMPUtilTest.bibtexDescription(bibtex)));
-
-        List<BibEntry> l = XMPUtil.readXMP(pdfFile.getAbsoluteFile(), xmpPreferences);
-        Assert.assertEquals(1, l.size());
-        BibEntry e = l.get(0);
-
-        Assert.assertNotNull(e);
-        Assert.assertEquals(Optional.of("OezbekC06"), e.getCiteKeyOptional());
-        Assert.assertEquals(Optional.of("2003"), e.getField("year"));
-        Assert.assertEquals(Optional.of("�pt�mz�t��n"), e.getField("title"));
-        Assert.assertEquals("misc", e.getType());
-    }
-
-    /**
-     * Make sure that the privacy filter works.
+     * The month attribute in DublinCore is the complete name of the month, e.g. March.
+     * In JabRef, the format is #mar# instead. To get a working unit test, the JabRef's
+     * bib-entry is altered from #mar# to {March}.
```

Comment:
Is there a reason why the month shouldn't be post-processed to get a proper bibtex value? We even have the `Month` class which handles parsing and converting to the correct output.